google_symptoms/params.json.template

```diff
@@ -1,8 +1,12 @@
 {
-  "export_start_date": "2020-02-20",
-  "static_file_dir": "./static",
-  "export_dir": "./receiving",
-  "cache_dir": "./cache",
-  "path_to_bigquery_credentials": "credentials.json",
-  "log_exceptions": false
+  "common": {
+    "export_dir": "./receiving",
+    "log_exceptions": false
+  },
+  "indicator": {
+    "export_start_date": "2020-02-20",
+    "num_export_days": 14,
+    "path_to_bigquery_credentials": "credentials.json",
```

Comment:
unrelated to this PR as a refactor, but why don't these appear in the ansible json?

Reply:
@nmdefries I think you may have added this line initially

Reply:
Yes, these params should also appear in the Ansible json. @sgsmob you can add them if you want or I'll make a followup PR.

Reply:
I will add it here, thanks for the info

Reply:
Done